packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -5,10 +5,11 @@ const { toBeAccessible } = require('../commands/expect');
 const { getViewports, describeViewports, setViewport } = require('../commands/viewport-helpers');
 
 const hideInputCaret = require('../commands/hide-input-caret');
+const { element } = require('../commands/validates');
 
 class TerraService {
   constructor(options = {}) {
-    this.formFactor = options.formFactor;
+    this.serviceOptions = options;
```

Comment:
This gets applied on the global browser? Nice

Reply:
Yes, the options object is added to `global.Terra`.